en/messages.json

```diff
@@ -76,9 +88,39 @@
             }
         }
     },
+    "popupRemainingAliases_2_mask": {
+        "message": "Remaining masks: $REMAINING$",
+        "description": "Tells the user how many more masks they can make.",
+        "placeholders": {
+            "remaining": {
+                "content": "$1"
+            }
+        }
+    },
+    "popupRemainingFreeMasks": {
+        "message": "Remaining free email masks:  $REMAINING$",
+        "description": "Tells the user how many more free masks they can make.",
+        "placeholders": {
+            "remaining": {
+                "content": "$1"
+            }
+        }
+    },
+    "popupLastUsedMasks": {
+        "message": "Last $LIMIT$ used masks",
```

Comment:
should we change it to the same format like `Last used masks: $LIMIT$` so the structure of this type of strings stay consistent?

Reply:
Heh. Just refactored it. :)